.github/workflows/ci.yml

```diff
@@ -24,6 +24,7 @@ jobs:
           - {use-orm: 0, use-dama: 1} # cannot happen
           # conflicts
           - {php: 8.0, symfony: 6.2.*}
+          - {php: 8.0, symfony: 6.3.*}
```

Comment:
not sure, but shouldn't we use "highest" deps with 6.3 now?

Reply:
Not exactly sure what you mean. This line just prevents php 8 and Symfony 6.3 as it can't be installed.